src/main/java/io/turntabl/Polls/Controllers/Controller.java

```diff
@@ -0,0 +1,176 @@
+package io.turntabl.Polls.Controllers;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.turntabl.Polls.dao.DAO;
+import io.turntabl.Polls.models.OptionTO;
+import io.turntabl.Polls.models.PollTO;
+import io.turntabl.Polls.models.ResponseTO;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.web.bind.annotation.*;
+
+@Api
+@RestController
+public class Controller implements DAO {
```

Comment:
> implements DAO

it shouldn't - your controller exposes your various REST endpoints, your DAOs (one for each table) expose database functionality. Right now anybody that uses an instance of your class has access to both REST endpoint functions and database functions.

Reply:
thanks, fixed :)